src/Bridges/vaftosafbridge.jl

```diff
@@ -0,0 +1,86 @@
+"""
+    VectorAffineBridge{T}
+
+The `VectorAffineBridge` splits a `VectorAffineFunction` into a series of
+`ScalarAffineFunction`s.
+"""
+struct VectorAffineBridge{S,T} <: AbstractBridge
+    constraints::Vector{CI{MOI.ScalarAffineFunction{T}, S}}
+end
+const VectorAffineBridgeSets = Union{MOI.Zeros, MOI.Nonpositives, MOI.Nonnegatives}
+function VectorAffineBridge{T}(model, f::MOI.VectorAffineFunction{T}, s::VectorAffineBridgeSets) where T
+    set_type = getscalarset(T, typeof(s))
+    sets      = [ set_type(-constant) for constant in f.constants ]
+    functions = [ MOI.ScalarAffineFunction{T}(MOI.ScalarAffineTerm{T}[], zero(T)) for term in f.terms ]
+    @assert dimension(s) == length(sets) == length(functions)
+    for term in f.terms
+        saf = functions[term.output_index]
+        push!(saf.terms, term.scalar_term)
+    end
+    ci = CI{MOI.ScalarAffineFunction{T}, S}[]
+    for (func, set) in zip(functions, sets)
+        push!(ci, MOI.addconstraint!(model, func, set))
+    end
+    VectorAffineBridge(ci)
+end
+getscalarset(T, ::Type{MOI.Zeros})        = MOI.EqualTo{T}
+getscalarset(T, ::Type{MOI.Nonpositives}) = MOI.LessThan{T}
+getscalarset(T, ::Type{MOI.Nonnegatives}) = MOI.GreaterThan{T}
+
+
+MOI.supportsconstraint(::Type{VectorAffineBridge{T,S}}, ::Type{MOI.VectorAffineFunction{T}}, ::Type{S}) where {T,S<:VectorAffineBridgeSets} = true
```

Comment:
S is different here. The first `S` is the scalar set and the second is the vector set